src/pds_doi_service/core/outputs/datacite/datacite_record.py

```diff
@@ -52,47 +52,74 @@ def __init__(self):
         with open(self._json_template_path, 'r') as infile:
             self._template = jinja2.Template(infile.read())
 
-    def create_doi_record(self, doi, content_type=CONTENT_TYPE_JSON):
+    def create_doi_record(self, dois, content_type=CONTENT_TYPE_JSON):
+        """
+        Creates a DataCite format DOI record from the provided list of Doi
+        objects.
+
+        Parameters
+        ----------
+        dois : Doi or list of Doi
+            The Doi object(s) to format into the returned record.
+        content_type : str, optional
+            The type of record to return. Only 'json' is supported.
+
+        Returns
+        -------
+        record : str
+            The text body of the record created from the provided Doi objects.
+
+        """
         if not content_type == CONTENT_TYPE_JSON:
```

Comment:
(Outside of the scope of the review, yes!) Why not ~~Zoidberg~~?

I mean, why not `!=`?